### ISSUE TITLE
Timeline for code/models?

### ISSUE THREAD

**Issue by segments-tobias** [NVlabs/VoxFormer#2]
Congrats on the paper, looks very interesting!
When do you plan on publishing the code?
Are you also going to publish any pre-trained models?

Thanks :)

**Comment by Chrisding** [comment]
We will publish both once the code review is done. Thank you for your interest!

**Comment by YoushaaMurhij** [comment]
Thanks for sharing this interesting work! 
I prefer to ask here rather than opening a new issue!
@Chrisding, I have a question about the performance of the model on certain devices. Did you measure the latency on any GPU device? Is this method suitable for embedded devices like Jetson Xavier?

Thanks, 
Looking forward the code and pertained models.

**Comment by TheWangYang** [comment]
@Chrisding 
### Congratulations on this article being successfully accepted by CVPR2023, I am very much looking forward to the code of this article and immersing myself in it!

**Comment by Chrisding** [comment]
We have released the code and pre-trained models. Thanks again for being interested in this work.
For the on device latency this is on our roadmap for next steps. We understand this is important. Will keep people updated.

**Comment by segments-tobias** [comment]
Thank you!
